apps/condo/domains/meter/hooks/useImporterFunction.ts

```diff
@@ -188,12 +192,12 @@ export const useImporterFunctions = (): [Columns, RowNormalizer, RowValidator, O
                 accountNumber: String(accountNumber),
                 number: String(meterNumber),
                 numberOfTariffs: parseInt(String(numberOfTariffs)),
-                verificationDate: dayjs(verificationDate).toISOString(),
-                nextVerificationDate: dayjs(nextVerificationDate).toISOString(),
-                installationDate: dayjs(installationDate).toISOString(),
-                commissioningDate: dayjs(commissioningDate).toISOString(),
-                sealingDate: dayjs(sealingDate).toISOString(),
-                controlReadingsDate: dayjs(controlReadingsDate).toISOString(),
+                verificationDate: validateDate(verificationDate),
+                nextVerificationDate: validateDate(nextVerificationDate),
+                installationDate: validateDate(installationDate),
+                commissioningDate: validateDate(commissioningDate),
+                sealingDate: validateDate(sealingDate),
+                controlReadingsDate: validateDate(controlReadingsDate) || dayjs().toISOString(),
```

Comment:
With this solution in case of invalid dates, a user will not see errors and will think, that everything is ok, — but dates in created `Meter` records will be `null`. To inform user about errors, you can check dates in `meterReadingValidator` and push error. These errors will be rendered to a new column in exported spreadsheet with not imported rows.

Reply:
Done